packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -403,125 +403,130 @@ export function updateAPIWithResolutionStrategyWithModels(cwd: string, settings:
-    } else {
-      const isFirstRestApi = settings.isFirstRestApi ?? true;
-      let chain = spawn(getCLIPath(), ['add', 'api'], { cwd, stripColors: true })
-        .wait('Select from one of the below mentioned services')
-        .send(KEY_DOWN_ARROW)
-        .sendCarriageReturn(); // REST
-
-      if (!isFirstRestApi) {
-        chain.wait('Would you like to add a new path to an existing REST API');
-
-        if (settings.path) {
-          chain
-            .sendConfirmYes()
-            .wait('Select the REST API you would want to update')
-            .sendCarriageReturn() // Select the first REST API
-            .wait('Provide a path')
-            .sendLine(settings.path)
-            .wait('Choose a lambda source')
-            .send(KEY_DOWN_ARROW)
-            .sendCarriageReturn() // Existing lambda
-            .wait('Choose the Lambda function to invoke by this path')
-            .sendCarriageReturn() // Pick first one
-            .wait('Restrict API access')
-            .sendConfirmNo() // Do not restrict access
-            .wait('Do you want to add another path')
-            .sendConfirmNo() // Do not add another path
-            .sendEof()
-            .run((err: Error) => {
-              if (err) {
-                reject(err);
-              } else {
-                resolve();
-              }
-            });
-          return;
-        } else {
-          chain.sendConfirmNo();
-        }
-      }
+    const isFirstRestApi = settings.isFirstRestApi ?? true;
+    let chain = spawn(getCLIPath(), ['add', 'api'], { cwd, stripColors: true })
+      .wait('Select from one of the below mentioned services')
+      .sendKeyDown()
+      .sendCarriageReturn(); // REST
 
-      chain.wait('Provide a friendly name for your resource to be used as a label for this category in the project');
-      if (settings.apiName) {
-        chain.sendLine(settings.apiName);
-      } else {
-        chain.sendCarriageReturn();
-      }
-      chain.wait('Provide a path').sendCarriageReturn().wait('Choose a lambda source');
+    if (!isFirstRestApi) {
+      chain.wait('Would you like to add a new path to an existing REST API');
 
-      if (settings.existingLambda) {
+      if (settings.path) {
         chain
-          .send(KEY_DOWN_ARROW)
+          .sendYes()
+          .wait('Select the REST API you want to update')
+          .sendCarriageReturn() // Select the first REST API
+          .wait('What would you like to do?')
+          .sendCarriageReturn() // Add another path
+          .wait('Provide a path')
+          .sendLine(settings.path)
+          .wait('Choose a lambda source')
+          .sendKeyDown()
           .sendCarriageReturn() // Existing lambda
           .wait('Choose the Lambda function to invoke by this path')
-          .sendCarriageReturn(); // Pick first one
+          .sendCarriageReturn() // Pick first one
+          .wait('Restrict API access')
+          .sendNo() // Do not restrict access
```

Comment:
just confirming you meant to change these from `sendConfirmNo` to `sendNo` (ie the prompts are using amplify-prompts)?